frontend/pages/shopping-lists/_id.vue

```diff
@@ -57,18 +57,21 @@
       <!-- View By Label -->
       <div v-else>
         <div v-for="(value, key) in itemsByLabel" :key="key" class="mb-6">
-          <div class="text-left">
-            <v-btn
-              :color="getLabelColor(value[0]) ? getLabelColor(value[0]) : '#959595'"
-              :style="{
-                'color': getTextColor(getLabelColor(value[0])),
-                'letter-spacing': 'normal',
-              }"
-            >
-              {{ key }}
-          </v-btn>
-          </div>
-          <v-divider/>
+          <span>
+            <div class="text-left">
+              <button @click="toggleShowLabel(key)">
+                <v-btn :color="getLabelColor(value[0]) ? getLabelColor(value[0]) : '#959595'">
+                  <v-icon>
+                    {{ labelOpenState[key] ? $globals.icons.chevronDown : $globals.icons.chevronRight }}
+                  </v-icon>
+                  {{ key }}
+                </v-btn>
+              </button>
+            </div>
+          </span>
```

Comment:
There are some unnecessairy things in here. 

```suggestion
          <v-btn 
            :color="getLabelColor(value[0]) ? getLabelColor(value[0]) : '#959595'"
            @click="toggleShowLabel(key)"
          >
            <v-icon>
              {{ labelOpenState[key] ? $globals.icons.chevronDown : $globals.icons.chevronRight }}
            </v-icon>
            {{ key }}
          </v-btn>
```